src/main/apiClient.ts

```diff
@@ -217,6 +221,52 @@ export default class ApiClient {
     })
   }
 
+  instantiateOneTap(opts: AuthOptions = {}) {
+    const script = document.createElement("script")
+    script.src = "https://accounts.google.com/gsi/client"
+    script.onload = () => this.googleOneTap(opts)
+    script.async = true
+    script.defer = true
+    document.querySelector("body")?.appendChild(script)
+  }
+
+  googleOneTap(opts: AuthOptions = {}) {
+    if (!isUndefined(this.config.googleClientId)) {
```

Comment:
What's the error handling in the SDK Core that could fit in the `else` of this structure?